### ISSUE TITLE
Distribution page layout

### ISSUE THREAD

**Issue by IronRico** [adiwg/mdEditor-doc#35]
This issue has a few components to address.
---

Taking a [look at the following section of the guide ](https://guide.mdeditor.org/reference/edit-window/metadata/distribution-panels/transferOption.html)first.

When looking at the left-hand navigation options for the guide, you will see the following structure.

![Ashampoo_Snap_Wednesday, June 17, 2020_10h46m49s_002_](https://user-images.githubusercontent.com/20647008/84919704-e6f34e80-b087-11ea-9bd5-2d8451aa240f.png)

Currently, there is an Online Transfer option that is not listed in that set above.  It should go above the Offline Transfer option. 

- Distribution Section
  - Distribution Edit Window
  - Distributor Edit Window
     - Transfer Option
        - **Online Transfer**
        - Offline Transfer
        - Distribution Format
- Order Process
---
As for the Online Transfer section, Here is the screenshot as it appears in the guide currently:

![image](https://user-images.githubusercontent.com/20647008/84920154-6a14a480-b088-11ea-9568-ef1af88765dd.png)

Here is how it appears in the current mdEditor version:

![image](https://user-images.githubusercontent.com/20647008/84920232-7ef13800-b088-11ea-941c-5d2c8913d97d.png)

---

I noticed that there are two parts that are different, those at the end. 
•	**Application Profile**
•	**Protocol Request**

These would need to be added to the Online Transfer page with relevant information.

**Application Profile** seems to just be the name of the API itself being used.  
**Protocol Request** seems in turn to just be the request time being made to the API, (POST, GET, PUT, or possibly even DELETE).  

---
Please provide any input you may have on this topic.

**Comment by IronRico** [comment]
I have requested a pull request on this, however, I did do some mock-up work in my own repo to see how this would look.  

Here is a screenshot of the Online Transfer option page.

![Ashampoo_Snap_Friday, June 19, 2020_21h16m54s_001_](https://user-images.githubusercontent.com/20647008/85189228-92f88d80-b272-11ea-97ee-284ea23ca4fc.png)
---

I would like input on the details of the bullet items, as to what proper wording would be wanted, ( or unwanted).

---

I also tested linking the page in the sidebar by editing the Summary.md file.

![Ashampoo_Snap_Friday, June 19, 2020_21h17m40s_002_](https://user-images.githubusercontent.com/20647008/85189238-cc30fd80-b272-11ea-9e65-72f6dfeea85a.png)

---

Please advise if this would have any negative issues using this method.

**Comment by IronRico** [comment]
**"Time Duration Object" or "Transfer Frequency"?**
---

On the Distribution page, I noted that there are these two sections that I assume are one in the same with different names.  I added screenshots below.

![Ashampoo_Snap_Monday, June 22, 2020_17h06m16s_001_](https://user-images.githubusercontent.com/20647008/85340080-0e448400-b4ab-11ea-932f-bc299fda46f6.png)


![Ashampoo_Snap_Monday, June 22, 2020_17h06m44s_002_](https://user-images.githubusercontent.com/20647008/85340094-1270a180-b4ab-11ea-9a62-6b9b3607b22e.png)

---

Are these indeed the same item only with different names, or are these totally separate?

**Comment by hamaier** [comment]
Nope, they're different. The attribute is "transfer frequency" which is a "time duration" object type. Just as the [Online Resource](https://guide.mdeditor.org/reference/edit-window/metadata/distribution-panels/transferOption.html#online-resource) "Name", "Protocol" and "Description" attributes are all of type "string".

**Comment by IronRico** [comment]
Second Update
---
Here is a screenshot of the Distribution section. I removed the Online Transfer and Time Duration Object items from the bottom and made them their own page. The links in-page work when tested on my localhost server as well.

![Ashampoo_Snap_Tuesday, June 23, 2020_18h28m49s_003_](https://user-images.githubusercontent.com/20647008/85477288-a875fc00-b57f-11ea-824e-a16637dd821d.png)


---
Here you can see the side navigation for those new areas.

![Ashampoo_Snap_Tuesday, June 23, 2020_18h28m06s_002_](https://user-images.githubusercontent.com/20647008/85477419-e541f300-b57f-11ea-9461-e1590f3d79db.png)



---

Here you can see the Online Transfer section.

![Ashampoo_Snap_Tuesday, June 23, 2020_18h29m28s_004_](https://user-images.githubusercontent.com/20647008/85477466-fd197700-b57f-11ea-950f-e9d8f5695cf6.png)



---

This is the Transfer Frequency page.

![Ashampoo_Snap_Tuesday, June 23, 2020_18h27m41s_001_](https://user-images.githubusercontent.com/20647008/85477492-0b679300-b580-11ea-8e65-c7f3bfacbbf3.png)



---

Please advise on the details of these items to help refine the tone or proper wording.

**Comment by IronRico** [comment]
Pull Request Review
---

Before submitting a pull request, I wanted to submit some of the changes I have made on the dev branch of my fork.  

---

Changed the "Summary.md" file to show the new page navigation options
under "Transfer Option".

These options are "Online Transfer Option" and "Transfer Frequency".

I created a page for Transfer Frequency. I used the same name for the file that 
was already in use, ("timeDuration-obj").

The path for this file is here:
mdEditor-doc/reference/edit-window/metadata/distribution-panels/timeDuration-obj.md

The page for the "Online Transfer Option" is found here:
mdEditor-doc/reference/edit-window/metadata/distribution-panels/onlineTransferOption.md

Edited the "Transfer Option" page located here:
mdEditor-doc/reference/edit-window/metadata/distribution-panels/transferOption.md

For that page, I removed the “Online Option” and “Time Duration” sections that were
found at the bottom.  

Updated the "Online Transfer Option" screenshot. It is located here:
assets/reference/edit-objects/metadata/distribution/OnlineTransferOption.png

**Comment by jlblcc** [comment]
This looks fine. Are you using the https://github.com/adiwg/mdEditor-doc/blob/master/reference/include-objects?

**Comment by IronRico** [comment]
I corrected the use of include-objects and also made sure the links in the Distribution section were working as well.

**Comment by IronRico** [comment]
Update 3

I worked to correct the navigation issues found in the Distribution section of the Summary.md file.

I replaced these with the following:

**[Online Transfer Option](reference/edit-window/metadata/distribution-panels/ onlineTransferOption.md)
[Transfer Frequency] (reference/edit-window/metadata/distribution-panels/ timeDuration-obj.md)**


I tested these items locally and found them to be working as intended.

This is noted in the rolling doc changes as well.
